docs/changes.rst

```diff
@@ -22,6 +22,9 @@ Released: not yet
 
 **Enhancements:**
 
+* Modify general help to display the full path of the default help file.
```

Comment:
Change "default help file" to "default connections file".

Reply:
DONE